src/components/shared/ActionBanner/index.scss

```diff
@@ -0,0 +1,39 @@
+.action-banner {
+  display: flex;
+  justify-content: flex-end;
+  align-items: flex-start;
+
+  &.usa-alert {
+      background-color: #f4e3db;
+
+    &::before {
+      width: 0;
+    }
+  }
+
+  &__icon {
+    display: flex-item;
+      margin-right: 13px;
+  }
+
+  &__button {
+    float: right;
+      display: flex-item;
```

Comment:
I don't think `display: flex-item` is a valid CSS rule. It can be removed, unless you had something else in mind.